src/multi-selector.js

```diff
@@ -64,7 +59,8 @@ const MultiSelector = React.createClass({
 			activeIndex: null,
 			searchValue: '',
 			close: (e) => {
-				if (!nearest(e.target, 'cpr-multi-selector')) {
+				const eventOccurredInsideOfThisComponent = this.el ? this.el.contains(e.target) : false;
```

Comment:
This is a better check because we want to close the dialogs of all pillboxes except for the pillbox that was clicked on.